invenio_rdm_records/resources/deserializers/rocrate/schema.py

```diff
@@ -0,0 +1,139 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022 CERN.
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""RO-Crate schema."""
+
+from distutils.log import error
+
+from flask_babelex import lazy_gettext as _
+from marshmallow import EXCLUDE, Schema, ValidationError, fields, pre_load, validate
+from marshmallow_utils.fields import SanitizedHTML, SanitizedUnicode
+
+
+def _list_value(lst):
+    """Return the first value from single-value lists."""
+    if isinstance(lst, list) and len(lst) == 1:
+        return lst[0]
+    return lst
+
+
+class ROCrateSchema(Schema):
+    """Schema for the loading RO-Crate into record metadata."""
+
+    class Meta:
+        """Meta attributes for the schema."""
+
+        unknown = EXCLUDE
```

Comment:
There are plenty of fields in RO-Crate about organization, metadata, schema versions, etc. (e.g. `@id`, `@type`, `@language`) which are important for formally validating an RO-Crate and JSON-LD, but for which we don't care about or use in our metadata. 

Making a list out of them and e.g. adding in `Meta.exclude` is also not possible since in JSON-LD one might just decide to add completely custom contexts (i.e. vocabularies) for storing arbitrary system/domain-specific metadata.